jetcd-core/src/main/java/io/etcd/jetcd/op/Op.java

```diff
@@ -29,8 +25,7 @@
 
 import com.google.protobuf.ByteString;
 
-import static io.etcd.jetcd.options.OptionsUtil.toRangeRequestSortOrder;
-import static io.etcd.jetcd.options.OptionsUtil.toRangeRequestSortTarget;
+import static io.etcd.jetcd.support.ProtoRequestMapper.*;
```

Comment:
please don't use wildcard import as it does not make it clear what is being imported

Reply:
ok, i see